package.json

```diff
@@ -53,15 +53,15 @@
     "@storybook/addons": "^3.4.8",
     "@storybook/react": "^3.4.8",
     "babel-core": "^6.26.3",
```

Comment:
Should this be updated too?

Reply:
I tried to upgrade babel to the latest version but it seems like there's not version of storybook that would allow upgrading babel.

@justafish Are you fine if we push this to a follow-up PR?

Reply:
Just looking at the output of yarn outdated, and want to talk about the scope of the follow-up.

There does seem to be a lot of red. I appreciate there is a balance between stability and being current. My way of thinking ... is that it is better to bite the bullet often and be responsive to issues that come up, rather than get stuck in intractable log jams.

For me material-ui is the stand out thing that is getting well behind the curve.

![outdated](https://user-images.githubusercontent.com/1259969/46652355-beb90a00-cb9a-11e8-854d-0cf6bac7bd5e.png)